tests/CMakeLists.txt

```diff
@@ -153,6 +153,11 @@ if(AVIF_ENABLE_GTEST)
     target_include_directories(avifreadimagetest PRIVATE ${GTEST_INCLUDE_DIRS})
     add_test(NAME avifreadimagetest COMMAND avifreadimagetest ${CMAKE_CURRENT_SOURCE_DIR}/data/)
 
+    add_executable(avifdecodetest gtest/avifdecodetest.cc)
```

Comment:
Nit: Note that the tests are listed in alphabetical order. Please move the new test up to the right place.

Reply:
Done.